src/export.rs

```diff
@@ -25,7 +26,15 @@ pub fn export(pin: &PinConfig, symlink_root: Option<&str>) -> Result<(), sysfs_g
         for name in &pin.names {
             let mut dst = path::PathBuf::from(symroot);
             dst.push(name);
-            try!(fs::symlink(format!("/sys/class/gpio{}", pin.num), dst));
+            try!(match fs::symlink(format!("/sys/class/gpio{}", pin.num), dst) {
+                Ok(_) => Ok(()),
+                Err(e) => {
+                    match e.kind() {
+                        ErrorKind::AlreadyExists => Ok(()),
```

Comment:
Instead of guessing that the existing symlink is just a previous version of the link, it would be safer to either 1) delete and recreate the symlink anyway or 2) check where the symlink goes.